conan/tools/meson/toolchain.py

```diff
@@ -189,6 +191,7 @@ def _add_apple_flags(self):
                 self.cpp_link_args += v
 
     def _context(self):
+        self.project_options.setdefault("wrap_mode", self.wrap_mode)
```

Comment:
why? Why not letting the interface be ``project_options``.

You can define in constructor ``self.project_options = {"wrap_mode": "nofallback"}`` and let users modify the dict if they want. This way we will not have to provide a explicit interface for each property

Reply:
Changed.